client/dashboard/domains/domain-transfer/transfer-domain-to-any-user.tsx

```diff
@@ -168,6 +183,87 @@ export default function TransferDomainToAnyUser() {
 		);
 	};
 
+	const renderConfirmationDialog = () => {
+		return (
+			<Modal
+				title={ __( 'Confirm Transfer' ) }
+				onRequestClose={ () => setIsTransferDialogOpen( false ) }
+			>
+				<VStack spacing={ 6 }>
+					<Text>
+						{ createInterpolateElement(
+							__( 'Do you want to transfer the ownership of <domainName/> to <recipientEmail/>?' ),
+							{
+								domainName: <strong>{ domainName }</strong>,
+								recipientEmail: <strong>{ formData.email }</strong>,
+							}
+						) }
+					</Text>
+					<HStack justify="flex-end" spacing={ 2 }>
```

Comment:
We have `ButtonStack` now to make the spacing consistent on HD

Reply:
Nice! Thanks :)